README.md

```diff
@@ -15,10 +20,11 @@ If you would like to get a head start with the tools we will be concentrating on
 * [Coordinate utilities](http://docs.astropy.org/en/stable/coordinates/index.html)
 * [Modeling and Fitting](http://docs.astropy.org/en/stable/modeling/index.html)
 * Interactive visualization and analysis tools:
-  * [Glue](http://glueviz.org/en/stable/)
-  * [imexam](http://imexam.readthedocs.io/en/latest/)
-  * spectroscopy tools such as [specview](https://github.com/spacetelescope/specview)
-  * Photometric tools, such as [photutils](http://photutils.readthedocs.io/en/latest/)
+  * [Glue](http://glueviz.org/)
+  * [imexam](http://imexam.readthedocs.io/)
+  * [Ginga](http://ginga.readthedocs.io)
+  * spectroscopy tools such as [specviz](https://github.com/spacetelescope/specviz)
```

Comment:
@stscieisenhamer @pllim  is this ready to go? I plan to merge today and email the participants